### DISCUSSION TITLE
Is there a way to view Volume Profile for larger time frames?

### DISCUSSION THREAD

**Discussion by Henulraj** [letianzj/QuantResearch#5] [Q&A]
Great work on Market Profile and Volume Profile!

Your market profile code is the most precise python code that anyone can find on the internet.

I was looking at the script and found that it only works on timeframes that have time as not constant, in other words, smaller timeframes.

Is there a possible way to make it work for larger time frames such as yearly timeframe as well?

Thanks again for the awesome work. 

<img width="1077" alt="Screen Shot 2021-08-08 at 8 04 22 AM" src="https://user-images.githubusercontent.com/81933094/128631360-60126ffb-fd88-4f91-84bd-8c2a2118e74f.png">

**Comment by letianzj** [comment]
Hi,

From the figure, I assume it is a daily bar of 1200 days? I'm not sure how good the flow is to the daily signal but technically it should work if df is the daily data frame.

The volume profile graph is composed of three parts. You already have the price (fig1) and volume (fig2), The only one missing is fig3 volume bar.  It just aggregates volume into price bars between cmin and cmax prices. You can set the return_raw=True to return the (x, y) of the bar to see if it is what you expect.

**Reply by Henulraj** [reply]
Thanks Letian for your reply! I'll check that and see if it works.
